spec/forms/steps/client/contact_details_form_spec.rb

```diff
@@ -11,10 +11,39 @@
 
   let(:telephone_number) { nil }
   let(:correspondence_address_type) { nil }
+  let(:has_home_address) { true }
```

Comment:
Should we set this to nil if the 'when applicant has home address' context explicitly sets it as true